### ISSUE TITLE
dhcp_server: do not log leases

### ISSUE THREAD

**Issue by lemoer** [freifunkh/ansible#70]
See here: https://pad.freifunk.net/p/nolog

```
freifunk logging abschalten


isc dhcp server

/etc/dhcpd.conf

# Use this to send dhcp log messages to a different log file (you also
# have to hack syslog.conf to complete the redirection).
log-facility local7;

/etc/rsyslog.conf

..

#
# First some standard log files.  Log by facility.
#
auth,authpriv.*                 /var/log/auth.log
*.*;auth,authpriv.none,local7.none      -/var/log/syslog
#cron.*                         /var/log/cron.log
daemon.*                        -/var/log/daemon.log
kern.*                          -/var/log/kern.log
lpr.*                           -/var/log/lpr.log
mail.*                          -/var/log/mail.log
user.*                          -/var/log/user.log

local7.* -/dev/null

...

# Some "catch-all" log files.
#
*.=debug;\
        auth,authpriv.none;\
        news.none;mail.none     -/var/log/debug
*.=info;*.=notice;*.=warn;\
        auth,authpriv.none;\
        cron,daemon.none;\
        mail,news.none
        local7.none     -/var/log/messages

.....
```

**Comment by 1977er** [comment]
Maybe --vacuum-time is our friend. Given that we do not need general logs otherwise, we can set this to 24h or 60s.

**Comment by lemoer** [comment]
I dislike the fact to have such short logs. Debugging things in the past is helpful, I guess. But on the other hand, we have Loki now. Is there any filtering before entries are filled into Loki?

**Comment by AiyionPrime** [comment]
We do?
Maybe we should tell us about like this.

**Comment by lemoer** [comment]
I just activated websockets for grafana, so live logging is now possible: [Click me Link](https://stats.ffh.zone/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Loki%22,%7B%22expr%22:%22%7Bhost%3D%5C%22sn10%5C%22%7D%22%7D,%7B%22mode%22:%22Logs%22%7D,%7B%22ui%22:%5Btrue,true,true,%22none%22%5D%7D%5D). The "Live"-button in the top-right of Grafana can be used to receive the live logs.

**Comment by lemoer** [comment]
**Further steps:**
- Deactivate logging of DHCP requests to Loki. Currently they are logged and publicly available :disappointed: 
    - [This article](https://www.bookstack.cn/read/loki/clients-promtail-scraping.md) describes how this could be done in the promtail config.
    - @1977er As you installed Loki without care, I guess this is now your job. :wink:
    - I just decided to stop Loki as of now to avoid exposing this sensitive information.
- Remove persistent journals/logs on the machines themselves?
    - Maybe we should consider keeping them in case Loki fails (or so).
    - Having them on the machine themselves might be convenient as well?

**Comment by AiyionPrime** [comment]
I'd vote for data minimization, and therefore nonpersistent logs.
Obviously Loki must not be made available for everyone -.-'

Please keep in mind for further testing of other tools, that the default configuration may very well harm the reputation of our project.

**Comment by 1977er** [comment]
>       * @1977er As you installed Loki without care, I guess this is now your job. 😉
>       * I just decided to stop Loki as of now to avoid exposing this sensitive information.

I do not see, where sensible information is leaked via Grafana. Unknown or newly created users cannot access the Loki datasource or edit dashboads. The given Loki example just exposes errors.

**Comment by lemoer** [comment]
Sorry, my mistake... Nevertheless I would suggest, that we do not forward dhcp logs to loki.

**Comment by 1977er** [comment]
Agreed.

**Comment by CodeFetch** [comment]
Implemented in PR #113
